### PR TITLE
prep/5.0.5

### PR DESCRIPTION
**Pull request by gamalielhere**
### Devop

- [x] Updated CHANGELOG.md
- [x] Add PR label

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +40,22 @@ MyEtherWallet is a doorway to the Ethereum blockchain, allowing users to manage
...
+6. App should be running in `https://localhost:8080`
+
+## Developers
+
+1. Open terminal
+2. Clone the repo: `git clone git@github.com:MyEtherWallet/MyEtherWallet.git`
+3. run `git checkout develop`
+4. run `npm i` to install node packages.
+5. run `npm run build`. You can also use the offline version by opening the index file from the dist folder with your preferred browser
+6. start `npm dev`
```
Comment (kvhnuke) [inline README.md]: this is `npm run dev`

File: src/containers/ConfirmationContainer/components/SuccessModal/SuccessModal.vue [hunk 2]
```diff
@@ -42,15 +43,35 @@ export default {
...
+    },
+    etherscanLink: {
+      type: String,
+      default: null
     }
   },
+  data() {
+    return {
+      buttonCheckEtherscan: {
+        title: 'Check Status on Etherscan',
```
Comment (kvhnuke) [inline src/containers/ConfirmationContainer/components/SuccessModal/SuccessModal.vue]: It is not always etherscan, we cant have it say `Check Status on Etherscan`

Comment (kvhnuke) [reply src/containers/ConfirmationContainer/components/SuccessModal/SuccessModal.vue]: may be simply say `Check Status` also, can you change all `etherscan` property names to `explorer` for example `etherscanLink` will become `explorerLink`

Comment (gamalielhere) [reply src/containers/ConfirmationContainer/components/SuccessModal/SuccessModal.vue]: incoming push should fix this to replace the name depending on network

**Pull request merged**